testdata/project-v4-config/PROJECT

```diff
@@ -34,6 +34,17 @@ resources:
   webhooks:
     conversion: true
     webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: testproject.org
+  group: crew
+  kind: FirstMate
+  path: sigs.k8s.io/kubebuilder/testdata/project-v4-config/api/v2
+  version: v2
+  webhooks:
+    conversion: true
+    webhookVersion: v1
```

Comment:
@momom-i,

See that we are now:

```
      $kb create webhook --group crew --version v1 --kind FirstMate --conversion --hub
      $kb create webhook --group crew --version v2 --kind FirstMate --conversion --spoke 
```

So, for  `--group crew --version v1 --kind FirstMate --conversion --hub` I would expect to have tracked here `hub:true`. And `spoke:true` for the second one. All that is the input we need to have tracked in the PROJECT config. So, for example, we can re-run all commands and re-scaffold the project from scratch by looking at it.

Reply:
I missed that. It is fixed at [91e74d7](https://github.com/kubernetes-sigs/kubebuilder/pull/3201/commits/91e74d7e484a842a5263db7344fb03d040b89f43)